frontend/src/pages/Player/Toolbar/DevToolsWindowV2/DevToolsWindowV2.tsx

```diff
@@ -86,12 +86,13 @@ const DevToolsWindowV2: React.FC<
 		(level) => level !== LogSourceValue.All,
 	) as unknown as LogSource[]
 
-	const form = useFormState({
+	const formStore = useFormStore({
 		defaultValues: {
 			search: '',
 		},
 	})
-	const filter = form.getValue(form.names.search)
+	const formState = formStore.getState()
```

Comment:
the dev tools filters require unfocusing to make the query

![image](https://github.com/highlight/highlight/assets/1351531/be72eed6-9af0-4580-8e12-4553b4b1b821)

Reply:
Fixed in [16d268f](https://github.com/highlight/highlight/pull/6417/commits/16d268ffbb810c398806b0450a490922385aeb10)